build.gradle

```diff
@@ -18,11 +18,28 @@ repositories {
 	// for more information about repositories.
 }
 
-minecraft {
```

Comment:
Why are you making this change over here?

Reply:
Not the author, but the `minecraft` scope is deprecated in Loom 0.10

Reply:
Thanks :)